.travis/before_install.sh

```diff
@@ -4,8 +4,8 @@ set -x
 
 if [ "$TRAVIS_OS_NAME" = "osx" ]; then
     brew install snap7
-    pip install nose mock coverage
-    pip3 install nose mock coverage
+    pip install nose mock coverage aiounittest
```

Comment:
i think this can be removed now since we dropped py2 support. the pip install in the script should work afaik.

Reply:
Will be removed on next commit.